### PR TITLE
[stable/kube-state-metrics] Add kubeconfig arg for kube-state-metrics 

### PR DESCRIPTION
**Pull request by mpritter76**
#### What this PR does / why we need it:
Added option to provide --kubeconfig argument to kube-state-metrics deployment and corresponding kube config secret. This allows us to set up an instance in one cluster to remote scape another cluster.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by mpritter76** [comment]
@mrueg Just wondering if there is something your waiting on for me? Been a couple weeks with no updates.

File: stable/kube-state-metrics/values.yaml [hunk 1]
```diff
@@ -130,6 +130,12 @@ collectors:
   verticalpodautoscalers: false
   volumeattachments: true
 
+# Enabling kubeconfig will pass the --kubeconfig argument to the container
+kubeconfig:
+  enabled: false
+  # base64 encoded kube-config file
+  secret: XXX
```
Comment (mrueg) [inline stable/kube-state-metrics/values.yaml]: let's leave it empty and don't add "XXX" here.

**Comment by mrueg** [comment]
@tariq1890 any thoughts on this one?

**Comment by mpritter76** [comment]
@tariq1890 @mrueg Is there anything left for me to do?

**Pull request merged**